source/deployment-guide/server/air-gapped-deployment.rst

```diff
@@ -1,428 +1,99 @@
+An air-gapped environment is one that is isolated from the public internet, requiring all necessary components to be available locally. This guide outlines what you'll need to deploy Mattermost in a self-hosted air-gapped environment, focusing on appropriate preparation, deployment guidance and configurations required for a successful deployment. 
 
-Deploying Mattermost in an air-gapped environment involves detailed preparation to ensure uninterrupted availability of critical components within the network. These efforts mitigate risks associated with the lack of internet connectivity by ensuring access to secure, localized resources. Key considerations include:
+Prerequisites
+-------------
 
-- Container images required when deploying in Kubernetes or Docker.
-- Mattermost server packages and dependencies required when deploying directly on Linux servers.
-- Access to a PostgreSQL database.
-- Object storage or a shared filesystem service for reliability accessing files from multiple Mattermost servers.
-- Other optional supporting services such as:
+Before disconnecting from the internet, you must gather all required packages, container images, and dependencies needed during the installation process. The resources you'll need will depend on the deployment method you are using, specifically:
 
-   - SMTP for email functionality.
-   - LDAP or SAML for authentication.
-   - Elasticsearch for performant post searching.
-   - Prometheus or other metric scraping services for enhanced observability.
+.. tab:: Linux
 
-Refer to Mattermost deployment documentation for setting up Mattermost as needed. This guide focuses on the supporting infrastructure needed for an air-gapped deployment which is required before deploying Mattermost in these environments.
+  Linux is recommeded as the simplest installation method for air-gapped environments. You can install the Mattermost Server in a few minutes on any air-gapped 64-bit Linux system using the tarball.
 
-Set up a self-hosted private container registry
---------------------------------------------------
+    **Bill of Materials**
 
-A private container registry securely stores Docker images for air-gapped deployments, ensuring compliance with data isolation requirements. Use it to enable local deployments in Kubernetes or Docker.
+    - :doc:`Mattermost tarball </product-overview/version-archive>`. We recommend using the latest :ref:`ESR <product-overview/release-policy:extended support releases>` for extended support where server upgrades may be infrequent)
+    - Database: PostgreSQL `installation packages <https://www.postgresql.org/download/>`_ or container images for your Linux distribution
+    - File Storage: 
+    - Load balancer: If you already have a load balancer you can skip this, otherwise you'll need 
```

Comment:
"otherwise we recommend Nginx.  https://docs.mattermost.com/deployment-guide/server/setup-nginx-proxy.html"